src/ampal/protein.py

```diff
@@ -555,26 +555,33 @@ def n_join(self, other, psi=-40.76, omega=-178.25, phi=-65.07,
         self.tags['assigned_ff'] = False
         return
 
-    def tag_sidechain_dihedrals(self, force=False, tag_rotamers=False):
```

Comment:
Removing the `tag_rotamers` option is breaking change so would require a major version bump. Is this necessary?

Reply:
Actually, I had added this option in this pr and then removed it to avoid breaking changes. 

The current code in master does not use `tag_rotamers` (see here: https://github.com/isambard-uob/ampal/blob/1f3dfbf152c2ff9556638bc8e44577e54dc9d4cc/src/ampal/protein.py#L558)